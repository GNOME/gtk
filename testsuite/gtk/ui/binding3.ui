<interface>
  <object class="GtkDropDown">
    <property name="expression">
      <constant>bla</constant>
    </property>
  </object>
</interface>
