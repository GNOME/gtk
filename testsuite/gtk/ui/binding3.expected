ERROR: gtk-builder-error-quark 13
.:0:0 Object with ID bla not found
